### PR TITLE
feat: scheduled deliveries message body

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/6858

### Description:

- [x] In the setup for my scheduled delivery, there is an `advanced options` collapse at the bottom that opens another section
- [x] If I open the toggle, I'm shown the option to `add message body` with a text box that takes markdown. (will markdown work in emails?)
- [x] This message is optional (i.e. it can be left blank).
- [x] If I add a message, then the message appears before any images or files in my email or in Slack. 
- [x] This message goes above the title + description that's currently included as text.

---

- [x] backend
  - [x] migration
  - [x] api routes + swagger
- [x] frontend
  - [x] custom md editor that works for slack + email
- [x] slack blocks
- [x] email templates

### CODE REVIEW

**Comment by IrakliJani** [comment]
dashboard csvs (no message/no limits):
CleanShot 2023-09-26 at 17 59 55@2x

---

dashboard csvs (with message/with limits):
CleanShot 2023-09-26 at 17 55 59@2x

---

chart csv (no message/no limits)
CleanShot 2023-09-26 at 18 00 49@2x

---

chart csv (with message/with limits)
CleanShot 2023-09-26 at 18 02 09@2x

---

chart image (with message)
CleanShot 2023-09-26 at 18 07 54@2x

File: packages/backend/src/scheduler/SchedulerTask.ts [hunk 1]
```diff
@@ -287,16 +292,14 @@ export const sendSlackNotification = async (
             imageUrl,
             csvUrl,
             csvUrls,
-            pdfFile, // TODO add pdf to slack
-        } =
-            notification.page ??
-            (await getNotificationPageData(scheduler, jobId));
+            // pdfFile, // TODO: add pdf to slack
```
Comment (ZeRego) [inline packages/backend/src/scheduler/SchedulerTask.ts:295]: forgotten todo ?

Comment (IrakliJani) [reply packages/backend/src/scheduler/SchedulerTask.ts:295]: @ZeRego it was there before

**Review by ZeRego — APPROVED** [review]

**Review by ZeRego — CHANGES_REQUESTED** [review]
- all Markdown blocks are dark mode now
Screenshot 2023-09-27 at 08 42 17

- different UI (link vs button) to view dashboard in app
Screenshot 2023-09-27 at 08 44 24
Screenshot 2023-09-27 at 08 43 46


- ~iirc, emails that were sent with 'send now' had a no scheduled delivery message at the bottom of the email~
  - ignore this, this was only changed for slack messages

**Comment by IrakliJani** [comment]
@ZeRego just fixed markdown dark mode
about point 2, I asked @PriPatel and it seems to be intentional.

> I think that was intended. Its not a new change. So when you're viewing CSVS, the primary CTA is to download. I dont want people to immediately click on the pick purple CTA. we can change it if anyone feels strongly! But its not a new change, basically.

**Review by ZeRego — APPROVED** [review]

**Pull request merged**